### PR TITLE
Crowdin (but we use it to test an auto-squash workflow)

### PR DESCRIPTION
**Pull request by TheBusyBot**

### CODE REVIEW

**Comment by TheBusyBiscuit** [comment]
Lets see if it will auto-squash this :o

File: src/main/resources/languages/lv/messages.yml [hunk 1]
```diff
@@ -35,7 +35,7 @@ guide:
     languages: 'Izvēlies savu vēlamo valodu,'
     credits: 'Slimefun4 Ieguldītāji'
   credits:
-    commit: "Ieguldījums"
+    commit: "\\\\t\nIeguldījums"
```
Comment (WalshyDev) [inline src/main/resources/languages/lv/messages.yml:38]: what is going on here...
```suggestion
    commit: "Ieguldījums"
```

**Review by WalshyDev — APPROVED** [review]

**Comment by TheBusyBiscuit** [comment]
Alright, good. It hasn't auto-squashed upon your review.

**Comment by TheBusyBiscuit** [comment]
Now my review should make it mergeable, here we go

**Review by TheBusyBiscuit — APPROVED** [review]

**Comment by TheBusyBiscuit** [comment]
Oh, it doesn't want to merge due to yaml linter failing XD

**Review by TheBusyBiscuit — DISMISSED** [review]
approve, now merge pls

**Comment by WalshyDev** [comment]
:eyes:

**Comment by TheBusyBiscuit** [comment]
Required label missing ("automerge")... I set that to "Translations Update" :NotLikeThis:

**Review by TheBusyBiscuit — DISMISSED** [review]
Will you work now?

**Review by TheBusyBiscuit — APPROVED** [review]
yo, work now pls. K thx

**Review by WalshyDev — APPROVED** [review]

**Comment by WalshyDev** [comment]
This action is so weird... it "skips" but then tries anyway. Can it merge though if it is a job which needs to complete? All jobs need to be complete in order to merge usually unless you admin force it so err, can it ever merge? LUL

**Comment by TheBusyBiscuit** [comment]
I honestly have no clue. I would really appreciate these to be auto-squashing themselves when someone approves them though.

**Review by TheBusyBiscuit — DISMISSED** [review]
test (I will query this via github's api to see more info.

**Review by TheBusyBiscuit — APPROVED** [review]

**Review by Sfiguz7 — APPROVED** [review]
There you go

**Comment by TheBusyBiscuit** [comment]
HELL YEAH!

**Pull request merged**